view/preview/index.js

```diff
@@ -15,7 +15,8 @@ const SELECTORS = {
 const CIRCLES = 30;
 const MIN_SCALE = 1.5;
 const MAX_SCALE = 4;
-const COLORS = ["#09b4fc", "#ffd201", "#01b302"];
+const COLORS = ["#00b4ff", "#ffd400", "#00b400"];
+// const COLORS = ["#fff000", "#00f000", "#00d2ff"];
```

Comment:
Lets remove this comment :)

if it is useful to us, then we will find it in the history of git